### PR TITLE
Switch license to LGPL3

### PR DESCRIPTION
**Pull request by goldyydev**
@HoloPanio and @amitojsingh366 need to consent to this **before this is merged** as they are currently away and their code is included in this library.

@TheOtterlord has given his consent to this license change in Discord

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+GNU Lesser General Public License
```
Comment (amitojsingh366) [inline LICENSE.md:1]: i am fine with it

Comment (amitojsingh366) [inline LICENSE.md:1]: i am fine with it

**Review by amitojsingh366 — APPROVED** [review]

**Comment by HoloPanio** [comment]
wtf is LGPL3

**Comment by goldyydev** [comment]
Read the discord

> wtf is LGPL3

**Review by HoloPanio — APPROVED** [review]

**Pull request merged**